static/styles/zulip.scss

```diff
@@ -320,6 +320,7 @@ input[type="email"],
 input[type="url"],
 input[type="search"],
 input[type="tel"],
+input[type="date"],
```

Comment:
This is already in this list above, and doesn't seem to fix the day mode color for that widget.

Reply:
got it. change a little on how it throws an exception if `CustomProfileField` is does not exist it throws an error but in case of `CustomProfileFieldValue` it pass. Does this make sense?